frontend/src/components/user-details/components/user-profile/components/user-profile-form/common.ts

```diff
@@ -7,4 +7,6 @@ const DEFAULT_UPDATE_USER_DETAILS_PAYLOAD: UserDetailsUpdateInfoRequestDto = {
   dateOfBirth: null,
 };
 
-export { DEFAULT_UPDATE_USER_DETAILS_PAYLOAD };
+const MIN_USER_AGE = 13;
```

Comment:
Extract this into shared, bcos mobile part also needs it. Don`t forget about MAX_USER_AGE